dev-util/dwarves/files/dwarves-1.17-musl.patch

```diff
@@ -0,0 +1,190 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2d23226..1798a96 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -74,6 +74,35 @@ if (NOT HAVE_REALLOCARRAY_SUPPORT)
+   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DCOMPAT_NEED_REALLOCARRAY")
+ endif()
+ 
++CHECK_C_SOURCE_COMPILES(
```

Comment:
Searching for the standalone version of obstack and argp seems like it is going to be pretty common when using musl. Is there a standard way to do this?